sentry_streams/src/messages.rs

```diff
@@ -129,6 +129,7 @@ pub struct PyAnyMessage {
     pub schema: Option<String>,
 }
 
+#[allow(dead_code)]
```

Comment:
https://github.com/getsentry/streams/blob/4ee04065320076478b07d265620286c0d3ed700b/sentry_streams/src/test_operators.rs#L39

I think this is not dead code